extensions/chghost-3.2.md

```diff
@@ -6,41 +6,47 @@ copyrights:
     name: "Christine Dodrill"
     period: "2013"
     email: "xena@yolo-swag.com"
```

Comment:
Please update this to me@christine.website, this email address is going to be discontinued.